common/src/main/java/com/microsoft/identity/common/internal/dto/AccessTokenRecord.java

```diff
@@ -81,8 +82,19 @@
          * String of kid. A thumbprint to an RSA keypair.
          */
         public static final String KID = "kid";
+
+        /**
+         * A hash of claims string sent to server that produced this AT.
+         */
+        public static final String REQUESTED_CLAIMS = "requested_claims";
     }
 
+    /**
+     * The JSON claims string sent to the server that produced this token. Used by MSAL C++
```

Comment:
Unclear to me from looking at this class whether or not this is a hashed value (or if it should be)

Reply:
Fixed the comment. There should be no hashing in AccessTokenRecord.java